src/v1/item/typers/map/typer.ts

```diff
@@ -56,13 +56,13 @@ export const map: MappedTyper = <
     _open,
     _savedAs,
     _default,
-    required: <$R extends RequiredOption = AtLeastOnce>(
-      nextRequired: $R = ('atLeastOnce' as unknown) as $R
+    required: <NextRequired extends RequiredOption = AtLeastOnce>(
```

Comment:
wdym by NextRequired?

Reply:
`required` is a function to set the next `required` state. So for instance:

```typescript
const attributeType = string() // => required option is "never"
  .required("always") // => required option becomes "always"
```

So it makes sense to name the type of "always" as `NextRequired`